.changeset/dirty-singers-promise.md

```diff
@@ -0,0 +1,6 @@
+---
+"@rocket.chat/meteor": patch
+"@rocket.chat/rest-typings": patch
```

Comment:
`patch` are meant for fixes.. since this is a new feature, it should be a `minor`